tests/test_session.py

```diff
@@ -202,7 +226,7 @@ def test_create_session_from_launch_fluent_by_passing_ip_and_port_and_password()
     )
     # check a few dir elements
     session_dir = dir(session)
-    for attr in ("field_data", "field_info", "setup", "solution"):
+    for attr in ("field_data", "field_info"):
```

Comment:
As this is testing the root/session object, shouldn't we be going in the other direction and making sure that more of the attributes are correctly exposed by `dir()`, rather than less? 

See also my other question. For example, I think the test should ideally be making sure that `file`, `mesh`, `results`, etc., are also exposed, in addition to `setup` and `solution` and the others; and not ignoring the fact that `setup` and `solution` aren't exposed anymore.

Reply:
Thanks, @raph-luc. It's important to have good coverage for this behaviour.

Reply:
@seanpearsonuk and @raph-luc, this test needs to be updated as it is using a static session data. New implementation makes a static info. call. That is the difference. Now, dir(solver) has the settings root methods like file, etc. exposed as well but in a live running session where valid settings api data is present and not a dummy session like the one used in this test.

Reply:
Got it, thanks for the explanation @prmukherj. 

With that in mind, I would say we should add a new test that instantiates a real Fluent `session` object, and checks if `dir(session)` indeed has all the attributes that we expect it to have.

In addition, as the test we are discussing here `test_create_session_from_launch_fluent_by_passing_ip_and_port_and_password` is not launching Fluent and is also no longer using `launch_fluent` (though I understand it did in the past), we should rename it to something like `test_session_from_mocked_fluent_using_ip_and_port_and_password`, as its current name can be misleading.